templates/views/message-item.html

```diff
@@ -19,7 +23,9 @@
                     <i title="Sent from {{mobile}}" class="icon mobile"></i>
                 {{/if}}
                 <i title="Reply to this message" class="f-reply icon link reply"></i>
-                <i title="Toggle detailed view" class="f-details-toggle icon link zoom"></i>
+                {{#if disableMessageInfo}}{{else}}
+                  <i title="Toggle detailed view" class="f-details-toggle icon link zoom"></i>
```

Comment:
NIT: spacing is inconsistent with surrounding context

Reply:
Yeah, my VS code is set to two spaces. Don't always catch it